lib/src/components/NxRadio/__tests__/NxRadio.test.tsx

```diff
@@ -21,146 +22,131 @@ describe('NxRadio', function() {
           <div className="bar"></div>
         </NxRadio>
       );
 
-      expect(render).toContainMatchingElement('label span.nx-radio__content .bar');
+      expect(container.querySelector('span.nx-radio__content .bar')).toBe;
     });
 
     it('does not render the .nx-radio__content element if there are no children', function() {
-      expect(getShallow()).not.toContainMatchingElement('.nx-radio__content');
+      const el = renderEl()!;
+      expect(el).not.toHaveClass('nx-radio__content');
     });
-
-    it('wraps the .nx-radio__content element in an NxOverflowTooltip unless overflowTooltip is set to false',
-        function() {
-          expect(getShallow()).not.toContainMatchingElement(NxOverflowTooltip);
-
-          expect(getShallow({ children: <div/> })).toContainMatchingElement(NxOverflowTooltip);
-          expect(getShallow({ children: <div/> }).find(NxOverflowTooltip))
-              .toContainMatchingElement('.nx-radio__content');
-
-          expect(getShallow({ children: <div/>, overflowTooltip: true }))
-              .toContainMatchingElement(NxOverflowTooltip);
-          expect(getShallow({ children: <div/>, overflowTooltip: true }).find(NxOverflowTooltip))
-              .toContainMatchingElement('.nx-radio__content');
-
-          expect(getShallow({ children: <div/>, overflowTooltip: null }))
-              .toContainMatchingElement(NxOverflowTooltip);
-          expect(getShallow({ children: <div/>, overflowTooltip: null }).find(NxOverflowTooltip))
-              .toContainMatchingElement('.nx-radio__content');
-
-          expect(getShallow({ children: <div/>, overflowTooltip: false }))
-              .not.toContainMatchingElement(NxOverflowTooltip);
-          expect(getShallow({ children: <div/>, overflowTooltip: false }))
-              .toContainMatchingElement('.nx-radio__content');
-        }
-    );
   });
 
-  it('calls its onChange prop with value argument when the input fires a change event', function() {
-    const onChange = jest.fn(),
-        component = getShallow({ onChange });
+  it('calls its onChange prop with value argument when the input fires a change event', async function() {
+    const onChange = jest.fn();
+    renderEl({ onChange });
+    const user = userEvent.setup();
 
     expect(onChange).not.toHaveBeenCalled();
-    component.find('input').simulate('change');
+    await user.click(screen.getByRole('radio'));
+
     expect(onChange).toHaveBeenCalledWith('red');
   });
 
   it('sets the input as readonly if there is no onChange handler', function() {
-    expect(getShallow().find('input')).toHaveProp('readOnly', false);
-    expect(getShallow({ onChange: undefined }).find('input')).toHaveProp('readOnly', true);
-    expect(getShallow({ onChange: null }).find('input')).toHaveProp('readOnly', true);
+    expect(renderEl()?.querySelector('input')).not.toHaveAttribute('readOnly');
+    expect(renderEl({ onChange: undefined })?.querySelector('input')).toHaveAttribute('readOnly');
```

Comment:
At the HTML level it's `readonly` (all lowercase)

Reply:
> At the HTML level it's `readonly` (all lowercase)

Updated here: d90926f741e3ce45781229b29afce572b46bf788